src/java/org/apache/cassandra/metrics/StorageMetrics.java

```diff
@@ -32,4 +34,13 @@ public class StorageMetrics
     public static final Counter exceptions = Metrics.counter(factory.createMetricName("Exceptions"));
     public static final Counter totalHintsInProgress  = Metrics.counter(factory.createMetricName("TotalHintsInProgress"));
     public static final Counter totalHints = Metrics.counter(factory.createMetricName("TotalHints"));
+
+    /* Tracks the most filled data_file_directory */
+    public static final Gauge<Double> diskUsed = Metrics.register(factory.createMetricName("DiskUsed"), new Gauge<Double>()
+    {
+        public Double getValue()
+        {
+            return Directories.getMaxPathToUtilization().getValue();
```

Comment:
Do we need to register the metric after a specific time laps? Or do we expect this value to be available as soon as C* starts up

Reply:
Doesn't really matter either way. The underlying method checks disk usage using the Java API for doing so, which should be something that can happen instantly.

Reply:
You're right, it should be fine. The `Directories` class populates the `dataDirectories` variable through a static block, and this variable is the one used to compute load